.vscode/codesnippets.code-snippets

```diff
@@ -0,0 +1,91 @@
+{
+	"Admonition: Caution": {
+		"prefix": "Caution",
+		"body": [
+			":::caution",
+			" ",
+			"$1",
+			" ",
+			":::"
+		],
+		"description": "Create a caution admonition."
+	},
+
+	"Admonition: Danger": {
+		"prefix": "Danger",
+		"body": [
+			":::danger",
+			" ",
+			"$1",
+			" ",
+			":::"
+		],
+		"description": "Create a danger admonition."
+	},
+
+	"Admonition: Info": {
+		"prefix": "Info",
+		"body": [
+			":::info",
+			" ",
+			"$1",
+			" ",
+			":::"
+		],
+		"description": "Create a information admonition."
+	},
+
+	"Admonition: Note": {
+		"prefix": "Note",
+		"body": [
+			":::note",
+			" ",
+			"$1",
+			" ",
+			":::"
+		],
+		"description": "Create a note admonition."
+	},
+
+	"Admonition: Tip": {
+		"prefix": "Tip",
+		"body": [
+			":::tip",
+			" ",
+			"$1",
+			" ",
+			":::"
+		],
+		"description": "Create a tip admonition."
+	},
+
+	"Admonition: Warning": {
+		"prefix": "Warning",
+		"body": [
+			":::warning",
+			" ",
+			"$1",
+			" ",
+			":::"
+		],
+		"description": "Create a warning admonition."
+	},
+
+	"Codeblock with Line Number": {
+		"prefix": "Code Block",
+		"body": [
+			"```bash showLineNumber",
+			"$1",
+			"```"
+		],
+		"description": "Create a shell code block with line numbers visible"
+	},
+
+	"Reading Time": {
```

Comment:
I think reading time is a docusaurus plugin, and I'd discourage adding it manually

Reply:
It's only a docusaurus plug-in on their blog posts. This is just a markdown code snippet to add these things I used to denote how long external pages would take to read in the "How to Install WSL" page.

![image](https://github.com/BobaBoard/boba-docs/assets/73132777/02617c03-6f5b-49cf-8fd6-205aa7dcb0ec)

Now that that page is complete, I can remove it, if we don't want to do time estimates on other things. We can also remove time estimates on the WSL page, but I received feedback that the page was intimidating with a lot of external links, so I added the fact that they're only a few minutes each to try to combat that.

Reply:
Ooooh makes sense. I think we can keep it then, but maybe just be explicit in the description like "add an inline reading time annotation for external links" (or something), so people know it's not the same functionality as the global plugin I was thinking about.

Reply:
Pushing this update now!